LiteCore/Database/DatabaseImpl+Upgrade.cc

```diff
@@ -110,6 +110,7 @@ namespace litecore {
             }
         }
         LogTo(DBLog, "\t%" PRIu64 " documents upgraded, now committing changes...", docCount);
+        DebugAssert(docCount == 0, "Should not upgrade all documents when openning the database.");
```

Comment:
Why is this here? The assertion will fail unless the database is empty or already uses version vectors.

Since you are commenting out the only call to this method, why not just delete it? Actually you can delete the whole source file, I think.

Reply:
Yes, I can simply delete that function. Perhaps I first #if it out and think to move it to cblite as a tool to upgrade db.

Reply:
Perhaps I can just delete it and file a ticket to move it to cblite that mentions the commit where the function is deleted.